recipes/swmmanywhere/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "swmmanywhere" %}
+{% set version = "0.1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/swmmanywhere-{{ version }}.tar.gz
+  sha256: 18690d31bf59a41edc2caada4302c47f41e7146c8e779f2ff0864788bcda68dc
+
+build:
+  skip: true  # [osx or py<310]
```

Comment:
Why isn't this noarch?

Reply:
Unfortunately, the `pyswmm` on `conda-forge` is very old and is not maintained anymore. Its osx version cannot be installed due to version conflict for `libopenmp` which is dep of `swmm`. I opened an issue on `pyswmm`'s feedstock and asked if they can update it. I am not sure if they will since it hasn't been updated since 4 years ago.

Reply:
I just opened a new issue in their main repo.

Reply:
So it would result in a non-solvable installation, right?

It is OK for noarch package to fails with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on the missing platform.

Reply:
Ah, I see. Makes sense. I will remove it.

Reply:
@ocefpaf per your suggestion, I added noarch. Builds except for osx pass.